gradle/wrapper/gradle-wrapper.properties

```diff
@@ -20,4 +20,4 @@ distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-4.10-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-6.9-bin.zip
```

Comment:
The all version is most of the time nicer for plugin development to get all sources etc.

Reply:
Got it.